### PR TITLE
Fixed #34194 -- Added django.utils.http.content_disposition_header().

### PR DESCRIPTION
**Pull request by alexmv**

### CODE REVIEW

File: docs/releases/4.1.txt [hunk 1]
```diff
@@ -427,6 +427,10 @@ Utilities
 
 * :func:`~django.utils.safestring.mark_safe` now preserves lazy objects.
 
+* The new :func:`~django.utils.http.content_disposition_header`
```
Comment (auvipy) [inline docs/releases/4.1.txt:430]: I think this should be moved to 4.2.txt

**Review by auvipy — CHANGES_REQUESTED** [review]

File: docs/ref/utils.txt [hunk 2]
```diff
@@ -729,6 +729,14 @@ escaping HTML.
 
     Outputs a string in the format ``Wdy, DD Mon YYYY HH:MM:SS GMT``.
 
+.. function:: content_disposition_header(as_attachment, filename)
+
```
Comment (felixxm) [inline docs/ref/utils.txt:733]: `versionadded:: 4.2` annotation is missing.

File: tests/utils_tests/test_http.py [hunk 3]
```diff
@@ -511,3 +512,28 @@ def test_rfc2231_wrong_title(self):
         for raw_line, expected_title in test_data:
             parsed = parse_header_parameters(raw_line)
             self.assertEqual(parsed[1]["title"], expected_title)
+
+
+class ContentDispositionHeaderTests(unittest.TestCase):
+    def test(self):
```
Comment (felixxm) [inline tests/utils_tests/test_http.py:518]:
```suggestion
    def test_basic(self):
```

File: docs/ref/utils.txt [hunk 4]
```diff
@@ -729,6 +729,14 @@ escaping HTML.
...
+    Constructs a ``Content-Disposition`` HTTP header value to match
+    the :rfc:`6266` specification.  Returns ``None`` if
+    ``as_attachment`` is ``False`` and ``filename`` is ``None``,
+    otherwise returns a string suitable for storing in the
+    ``Content-Disposition`` header.
```
Comment (felixxm) [inline docs/ref/utils.txt:738]: Wrap at 79 chars.

**Comment by alexmv** [comment]
Thanks for the reviews, @auvipy and @felixxm!  Fixed and repushed.

File: docs/ref/utils.txt [hunk 5]
```diff
@@ -729,6 +729,15 @@ escaping HTML.
...
     Outputs a string in the format ``Wdy, DD Mon YYYY HH:MM:SS GMT``.
 
+.. function:: content_disposition_header(as_attachment, filename)
+
+    .. versionadded:: 4.2
+
+    Constructs a ``Content-Disposition`` HTTP header value to match the
+    :rfc:`6266` specification.  Returns ``None`` if ``as_attachment`` is
+    ``False`` and ``filename`` is ``None``, otherwise returns a string suitable
+    for storing in the ``Content-Disposition`` header.
```
Comment (shangxiao) [inline docs/ref/utils.txt:739]: What do other think about putting this in the docstring as well?

Comment (felixxm) [reply docs/ref/utils.txt:739]: I pushed small edits to docs and `content_disposition_header()`'s docstring.

**Review by felixxm — APPROVED** [review]
@alexmv Thanks :+1: Welcome aboard :boat: 

I pushed small edits.

**Pull request merged**